DataFormats/ParticleFlowReco/src/classes_serial_def.xml

```diff
@@ -1,29 +1,29 @@
 <lcgdict>
+  <class name="reco::CaloRecHitHostCollection"/>
+  <!-- reco::CaloRecHitHostCollection::Layout must be listed before the aliased-to type -->
+  <class name="reco::CaloRecHitHostCollection::Layout"/>
   <class name="reco::CaloRecHitSoA"/>
   <class name="reco::CaloRecHitSoA::View"/>
-  <class name="reco::CaloRecHitHostCollection" ClassVersion="3">
-    <version ClassVersion="3" checksum="1876594952"/>
-  </class>
```

Comment:
I also removed the explicit versions and checksums for the `PortableHostCollection`s that still had it. Their versioning should be done in a different way because `PortableHostCollection<T>` is a class template.